.gitignore

```diff
@@ -24,6 +24,7 @@ precompiled
 /.project
 project/project
 project/target
+project/metals.sbt
```

Comment:
I'm not wedded to adding this into the gitignore, but I do like to occasionally use vs-code with metals for sbt support 🤷🏻 If that's a no-go, I can remove this!